assets/src/setup/pages/save/index.js

```diff
@@ -31,26 +114,85 @@ export function Save() {
 
 	/**
 	 * Triggers saving of user options on arrival of this screen.
-	 *
-	 * @todo Possibly wait for a different user action to save.
 	 */
 	useEffect( () => {
 		if ( ! didSaveDeveloperToolsOption && ! savingDeveloperToolsOption ) {
 			saveDeveloperToolsOption();
 		}
 	}, [ didSaveDeveloperToolsOption, savingDeveloperToolsOption, saveDeveloperToolsOption ] );
 
-	if ( savingOptions || savingDeveloperToolsOption ) {
-		return <Loading />;
+	if ( shouldShowSaving || savingOptions || savingDeveloperToolsOption ) {
+		return <Saving />;
 	}
 
-	if ( ! didSaveOptions || ! didSaveDeveloperToolsOption ) {
-		return null;
+	let heading = __( 'Congratulations!', 'amp' );
+	if ( 'standard' === themeSupport ) {
+		heading = __( 'Your site is ready', 'amp' );
 	}
 
 	return (
-		<div>
-			{ __( 'Options saved', 'amp' ) }
+		<div className="done">
+			<div className="done__text">
+				<h1>
+					{ heading }
+				</h1>
+				<p>
+					{ getDescription( themeSupport ) }
+				</p>
+			</div>
+			<div className="done__preview-container">
+				<AMPInfo icon={ ( props ) => <IconMobile { ...props } /> }>
+					{ __( 'Live view of your site', 'amp' ) }
```

Comment:
Should we explicitly state that the preview is an AMP page? Otherwise, it may confuse users as to which version they are looking at (especially when in reader/transitional mode):

![image](https://user-images.githubusercontent.com/16200219/86258099-f4412a80-bba9-11ea-835e-8833800ceee0.png)


```suggestion
					{ __( 'Live view of your AMP site', 'amp' ) }
```

Reply:
Done in 05ae9e993f8a3551eb55cf696d839022754ed5d9